redbot/cogs/streams/streams.py

```diff
@@ -760,6 +769,11 @@ async def _get_mention_str(self, guild: discord.Guild) -> Tuple[str, List[discor
         if await settings.mention_here():
             mentions.append("@here")
         can_manage_roles = guild.me.guild_permissions.manage_roles
+        can_mention_everyone = guild.me.guild_permissions.mention_everyone
```

Comment:
It would be better if this were checking for permissions in channel:
```suggestion
        can_mention_everyone = channel.permissions_for(guild.me).mention_everyone
```
This requires you to also add a `channel` argument to this function and pass it in the place where it's called.